translator/GCP_cloud_function/README.md

```diff
@@ -0,0 +1,18 @@
+# iCone translator Cloud Function
+
+A system was created in google cloud platform to automatically translate iCone data to WZDx message.This system consits of two buckets and a cloud function.
+When a file is uploaded to the first bucket, the cloud function automatically downloads the file, translates to a WZDx message and uploads it to the second bucket.
+
+![alt text](https://github.com/CDOT-CV/Work_Zone/blob/dev/translator/GCP_cloud_function/iCone%20Translator%20block%20diagram.png)
+
+## files present
+cloud_function.zip
```

Comment:
Instead of a zip file, let's add these files in uncompressed so we can review them and push changes as needed.

Reply:
I removed the Zip file and added the uncompressed files and updated the README.md description